### PR TITLE
Package macaque.0.8

### PR DESCRIPTION
**Pull request by jrochel**
### `macaque.0.8`
DSL for SQL Queries in Caml

---
* Homepage: https://github.com/ocsigen/macaque
* Source repo: git+https://github.com/ocsigen/macaque.git
* Bug tracker: https://github.com/ocsigen/macaque/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/macaque/macaque.0.8/opam [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+maintainer: "gabriel.scherer@gmail.com"
+homepage: "https://github.com/ocsigen/macaque"
+bug-reports: "https://github.com/ocsigen/macaque/issues"
+dev-repo: "git+https://github.com/ocsigen/macaque.git"
+build: [
+  ["./configure" "--prefix" prefix]
+  [make]
+]
+install: [ make "install" ]
+remove: [ "ocamlfind" "remove" "macaque" ]
```
Comment (kit-ty-kate) [inline packages/macaque/macaque.0.8/opam]:
```suggestion
```
This field is unnecessary since opam 2.0

File: packages/macaque/macaque.0.8/opam [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+depends: [
+  "ocaml"
+  "ocamlfind" {build}
+  "pgocaml" {>= "2.2"}
+  "oasis" {>= "0.4.4"}
+  "camlp4"
+]
+authors: "gabriel.scherer@gmail.com"
+flags: light-uninstall
```
Comment (kit-ty-kate) [inline packages/macaque/macaque.0.8/opam]:
```suggestion
```
This field can be removed as soon as the `remove` field will be removed.

**Comment by XVilka** [comment]
Please limit the OCaml version, since 4.09 doesn't support camlp4: https://github.com/ocsigen/macaque/issues/40

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**